README.md

```diff
@@ -207,20 +207,20 @@ The order in which you handle effects in Kyo can significantly influence both th
 import kyo._
 import scala.util._
 
-def optionsFirst(a: Int < (Options & Aborts[Exception])): Either[Exception, Option[Int]] = {
+def optionsFirst(a: Int < (Options & Aborts[Exception])): Either[Exception, Option[Int]] = 
   val b: Option[Int] < Aborts[Exception] = 
     Options.run(a)
   val c: Either[Exception, Option[Int]] < Any = 
     Aborts.run[Exception](b)
   c.pure
-}
```

Comment:
Thanks for picking this up! Please take a look at Kyo's codebase to follow the same formatting. For example, method bodies with more than 4 lines have an [end maker](https://github.com/getkyo/kyo/blob/326f7588402cbe0a86133615db98b8c9cafed3bd/.scalafmt.conf#L8). I wonder if there'd be a way to make scalafmt format the code in the readme since it'd be more future-proof.

Reply:
I think this bounty should involve configuring scalafmt on the readme.

Reply:
@mobley-trent let us know if you'd be ok with the scope change. I could also increase the bounty.

Reply:
@fwbrasil it's perfectly fine. I'm currently looking into how I can integrate scalafmt with mdoc during the build process.

Reply:
@mobley-trent You could put the code in scala files and scalafmt will format them anyway. And read in the files. We do this in zio projects. For example like this.
https://github.com/zio/zio-http/blob/342ac4b578a5d7fd11229bcea3b546f632ff67b8/zio-http-docs/src/main/scala/utils.scala#L39

Reply:
@fwbrasil I would appreciate your feedback